doc/classes/Node.xml

```diff
@@ -43,14 +43,6 @@
 				Call [method update_configuration_warnings] when the warnings need to be updated for this node.
 			</description>
 		</method>
-		<method name="_get_configuration_warnings" qualifiers="virtual">
-			<return type="String[]" />
-			<description>
-				The elements in the array returned from this method are displayed as warnings in the Scene Dock if the script that overrides it is a [code]tool[/code] script.
-				Returning an empty array produces no warnings.
-				Call [method update_configuration_warnings] when the warnings need to be updated for this node.
-			</description>
-		</method>
```

Comment:
What happened to this method description? It's being completely removed.

Reply:
The function existed twice due to a mistake from my part. The actual one is above this.